app/content/plan/index.js

```diff
@@ -1,14 +1,205 @@
-import { PaperProvider, Text } from "react-native-paper";
+import React, { useState } from 'react';
 import { SafeAreaView } from "react-native-safe-area-context";
+import { PaperProvider, Text, Avatar, Button, Card, Appbar } from "react-native-paper";
+import { TouchableOpacity } from 'react-native';
+import { FlatList, StyleSheet, ScrollView, RefreshControl } from 'react-native';
+import Icon from "react-native-vector-icons/MaterialCommunityIcons";
+import { View } from 'react-native';
+import { useNavigation, router } from 'expo-router';
+import { Link } from 'expo-router';
+import drillsData from "~/drill_data.json";
+import { collection, doc, getDoc, getDocs, query, updateDoc, where } from "firebase/firestore";
+import { db } from "~/firebaseConfig";
+import { useEffect } from 'react';
+
+import { formatDate, getUnique } from '~/Utility'
+import { useDrillInfo } from "~/hooks/useDrillInfo";
+import { useUserInfo } from "~/hooks/useUserInfo";
+import Loading from "~/components/loading";
+import { currentAuthContext } from "~/context/Auth";
+
+import { useQueryClient } from '@tanstack/react-query';
+
+
+const DrillList = () => {
+  const queryClient = useQueryClient();
+  const { currentUserId, currentTeamId } = currentAuthContext();
+  const userId = currentUserId ?? null;
+  const teamId = currentTeamId ?? null;
+
+  console.log("USER ID IN PLAN", userId)
+
+  const {
+    data: drillInfo,
+    error: drillInfoError,
+    isLoading: drillInfoIsLoading,
+  } = useDrillInfo();
+
+  const {
+    data: userData,
```

Comment:
userInfo. Try to match the hook you're calling. userData should be any other data, something like
userData = userInfo["assignedData"]